client/src/components/navbar/Navbar.scss

```diff
@@ -0,0 +1,6 @@
+.navbar {
+    background-color: rgb(197, 197, 197);
+    display: flex;
+    height: 4em;
```

Comment:
We can change that to a css variable here and at the App.scss so no style bugs would happen